etc/multivm_bootloaders/vm_evm_emulator/commit

```diff
@@ -0,0 +1 @@
+900e344006db93b149c888f21596941d768497c3
```

Comment:
it doesn't match actual commit used 00946002e684966eab680f919c8c517f8cc64d9e

Reply:
Regenerated, bytecodes are the same